src/utils.ts

```diff
@@ -321,21 +319,15 @@ export async function askAndOpenInRemote(
   if (result === DialogResponses.yes) {
     telemetryContext.properties.errorMessage =
         `${operation} Operation failed and user opens project in container.`;
-    const res = await RemoteExtension.checkRemoteExtension(channel);
-    if (!res) {
-      telemetryContext.properties.errorMessage =
-          `${operation} operation failed on installing Remote Extension.`;
-      return false;
-    }
+    await RemoteExtension.checkRemoteExtension();
+
     await vscode.commands.executeCommand('openindocker.reopenInContainer');
-  } else {
-    const message = `${operation} can only be executed in remote container.`;
-    channelShowAndAppendLine(channel, message);
-    telemetryContext.properties.errorMessage = `${operation} operation failed.`;
   }
 
-  return false;
+  throw new CancelOperationError(`${
```

Comment:
Since the function return void, does it mean that it will always throw error here?

Reply:
No need to throw cancel operation error here. Fixed.